execution_workflows/APA-Scan/APA-Scan.py

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
This looks like the APA-Scan code itself, no? If so, I think this should be put inside a Docker container, copied straight from its original repo, no need to keep a copy here. Goes for all other APA-Scan scripts as well, e.g., `make_plots.py`, `methods.py` etc.

You can check out the Dockerfile in the mountainClimber PR to get a rough idea on how to do that (by cloning the Git repo and then copying the Python scripts): https://github.com/iRNA-COSI/APAeval/blob/ewf_mountainclimber/execution_workflows/mountainClimber/mountainClimber_nextflow_dsl2/docker/Dockerfile_method